src/server/qgsfcgiserverrequest.cpp

```diff
@@ -211,7 +211,22 @@ void QgsFcgiServerRequest::printRequestInfos( const QUrl &url )
     QStringLiteral( "NO_PROXY" ),
     QStringLiteral( "HTTP_AUTHORIZATION" ),
     QStringLiteral( "QGIS_PROJECT_FILE" ),
-    QStringLiteral( "QGIS_SERVER_IGNORE_BAD_LAYERS" )
+    QStringLiteral( "QGIS_SERVER_IGNORE_BAD_LAYERS" ),
+    QStringLiteral( "SERVICE_URL" ),
+    QStringLiteral( "WMS_SERVICE_URL" ),
+    QStringLiteral( "WFS_SERVICE_URL" ),
+    QStringLiteral( "WMTS_SERVICE_URL" ),
+    QStringLiteral( "WCS_SERVICE_URL" ),
+    QStringLiteral( "HTTP_X_SERVICE_URL" ),
+    QStringLiteral( "HTTP_X_WMS_SERVICE_URL" ),
+    QStringLiteral( "HTTP_X_WFS_SERVICE_URL" ),
+    QStringLiteral( "HTTP_X_WMTS_SERVICE_URL" ),
+    QStringLiteral( "HTTP_X_WCS_SERVICE_URL" ),
+    QStringLiteral( "HTTP_FORWARDED" ),
+    QStringLiteral( "HTTP_X_FORWARDED_HOST" ),
+    QStringLiteral( "HTTP_X_FORWARDED_PROTO" ),
+    QStringLiteral( "HTTP_HOST" ),
+    QStringLiteral( "SERVER_PROTOCOL" )
```

Comment:
I think that QGIS_SERVER_ should be used as a prefix for homogeneity.

Reply:
For the Server URL environment variable :-)

Reply:
Yes :)